src/main/drivers/dma.h

```diff
@@ -22,13 +22,15 @@ typedef void (*dmaCallbackHandlerFuncPtr)(struct dmaChannelDescriptor_s *channel
 #ifdef STM32F4
 
 typedef enum {
-    DMA1_ST1_HANDLER = 0,
+    DMA1_ST0_HANDLER = 0,
```

Comment:
HAHA - I just found this one, and the missing DMA2_Stream6 & 7 handlers for sorting out telemetry! ;)